pkg/proxy/iptables/proxier.go

```diff
@@ -394,6 +394,8 @@ var iptablesKubeletJumpChains = []iptablesJumpChain{
 // on upgrade.
 var iptablesCleanupOnlyChains = []iptablesJumpChain{}
 
+var conntrackStateInvalidDropRule = fmt.Sprintf("-A %s -m conntrack --ctstate INVALID -j DROP", kubeForwardChain)
```

Comment:
How about you add a comment to the rule, and scan for that instead?

(Especially, there have been situations where you add a rule like `-A CHAIN <foo> <bar> -j DROP` and when you list it back, it shows up as `-A CHAIN <bar> <foo> -j DROP`. In this case the rule is simple enough that there's not much opportunity for iptables to rewrite it, but it's still best to not assume, on general principles.)